htmx.ext.shoelace.js

```diff
@@ -1,41 +1,49 @@
-const slTypes = 'sl-button, sl-checkbox, sl-color-picker, sl-input, sl-radio-group, sl-range, sl-rating, sl-select, sl-switch, sl-textarea'
+const slTypes = "sl-button, sl-checkbox, sl-color-picker, sl-input, sl-radio-group, sl-range, sl-rating, sl-select, sl-switch, sl-textarea";
 
 /* Lightly modified version of the same function in htmx.js */
 function shouldInclude(elt) {
-
 	// sl-rating doesn't have a name attribute exposed through the Shoelace API
-	if (elt.tagName == 'SL-RATING' && elt.getAttribute('name')) {
+	if (elt.tagName === "SL-RATING" && elt.getAttribute("name")) {
 		return true;
 	}
-	if (elt.name === "" || elt.name == null || elt.disabled) {
```

Comment:
Why did you remove the `elt.disabled` - did you find it was unnecessary?

Reply:
yes, you are right:
Tip: Disabled <input> elements in a form will not be submitted! https://www.w3schools.com/tags/att_input_disabled.asp

Reply:
@benopotamus